.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,58 @@
+name: "🐛 Bug Report"
+description: "If something isn't working as expected 🤔."
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
+
+  - type: textarea
+    attributes:
+      label: What is the issue?
+      description: A clear and concise description of what kube is doing and what you would expect.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: How can it be reproduced?
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Logs, error output, etc
+      description: If the output is long, please create a [gist](https://gist.github.com/) and paste the link here.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: output of `linkerd check`
```

For a while now the full output of `linkerd check` has not been too helpful in issue reports; its there so that we know the user has ran it themselves. Therefore, I recommend we change this to `-o short` so that issue reports are a little shorter.

```suggestion
      label: output of `linkerd check -o short`
```